sdk/include/asm/ksx.template.h

```diff
@@ -779,7 +779,7 @@ OFFSET(ThSpecialApcDisable, KTHREAD, SpecialApcDisable),
 //OFFSET(ThVfpState, KTHREAD, VfpState),
 #endif
 OFFSET(ThNextProcessor, KTHREAD, NextProcessor),
-OFFSET(ThProcess, KTHREAD, Process),
+//OFFSET(ThProcess, KTHREAD, Process),
```

Comment:
Why does this need to be disabled?

Reply:
Because it's defined twice.

Reply:
```
R:\build\dev\x86_arm\sdk\lib\pseh\asm\seh_prolog_pseh_asm.asm(1198) : error A2026: multiple symbol definition or incompatibility: ThProcess
```
From: https://chat.reactos.org/reactos/pl/dx13gdrhxtgcfbq7ckdy7s1krr

Reply:
Fully remove the line, then?